### PR TITLE
Added content per request of team talent and add'l contributors 

### PR DESCRIPTION
**Pull request by elainekamlley**
Still a WIP but added:  
- navigation tabs to follow the process of applying to 18F: Intro, Who we are hiring, How to apply, Interview process.
- Updated the "How to apply" section to go to USDS application. 
- positions that are included in the Agile BPA descriptions as per request from @jamiealbrecht 
- Adjusted the interview section. Basic criteria is mid level and edited specific interview details for engineering and product to follow at the bottom.

### CODE REVIEW

**Comment by afeld** [comment]
Are putting the position descriptions here in lieu of https://pages.18f.gov/agile-labor-categories/? We probably don't need them in both places.

/cc @gboone @cscairns

**Comment by elainekamlley** [comment]
Does https://pages.18f.gov/agile-labor-categories/ serve different purpose? Is this page shared in conjunction with Agile BPA and supporting clients? If it does, is it more helpful to have that language separate from the rest of 18F hiring content? 

I talked with @jamiealbrecht that these descriptions are being sent to potential candidates as well and confirmed with her that these should be the descriptions on our site. I started with building content from the https://18f.gsa.gov/2015/02/25/We-Are-Hiring/ but didn't want to confuse candidates with language on our site and language being sent from our staff.

**Comment by cscairns** [comment]
https://pages.18f.gov/agile-labor-categories/ this is specific to labor categories we suggest for agile procurements...we've been using it as an interim reference for recruiting / hiring, but that will change soon

File: pages/benefits.md [hunk 1]
```diff
@@ -16,3 +16,20 @@ There is also [Life Insurance](http://www.opm.gov/healthcare-insurance/life-insu
...
+Government's version of a 401K offers up to 5 percent matching: [Thrift Savings Plan](https://www.tsp.gov/)
+
+
+### Teleworking
+
+Telework provides flexibility for an employee. It improves the morale and productivity of staff by allowing for greater collaboration amongst peers. 18F recognizes these benefits and encourages working from home with the approval of their administrative supervisor or his/her designee. This policy adheres to and is in conjunction with [GSA's Mobility and Telework Policy](http://www.gsa.gov/graphics/staffoffices/GSAteleworkpolicy.pdf).
+
+### Commuter Benefits
+
+
```
Comment (afeld) [inline pages/benefits.md]: Maybe just add a `TODO` for us to add a link later?

File: pages/how-to-apply.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+permalink: /how-to-apply/
+layout: default
+title: How to apply
+---
+
+Submit your application through [U.S. Digital Services](https://www.whitehouse.gov/digital/united-states-digital-service/apply). Enter "join18F" as the referral code. There's no need for a cover letter; within three weeks our team will review your application and contact you about next steps. Check out more information about our interview process [here](/joining-18f/interview-process).
+
+### Tips on resume preparation
+
+
```
Comment (afeld) [inline pages/how-to-apply.md]: Add `TODO`.

File: pages/interview-process.md [hunk 3]
```diff
@@ -4,9 +4,11 @@ layout: default
 title: The 18F Interview Process
 ---
 
-### First Steps
+### Phone screen
+
+After you submit your application through [U.S. Digital Services](https://www.whitehouse.gov/digital/united-states-digital-service/apply) within three weeks, our team will review your resume. If it seems like you are a good fit for 18F, we will reach out to schedule a 30-minute phone call. On the phone call, plan to talk more about your skills and experience, what you're passionate about, and the work we do here at 18F.
```
Comment (afeld) [inline pages/interview-process.md]: First sentence can be rephrased

> Within three weeks of you [submitting your application](/how-to-apply/), our team will review your resume.

File: pages/interview-process.md [hunk 4]
```diff
@@ -57,24 +37,61 @@ After your interview, the interviewer(s) will evaluate you along the basic rubri
...
+There isn't one description that matches the role of all Product Leads at 18F. Ultimately, it depends on the nature of the relationship between 18F and a partner agency.  Some relationships have one account manager with multiple product managers who focus on building out separate products while others have one person who is an Account Manager/Product Manager hybrid. No matter where you fall on the spectrum between Account Manager, Product Manager (and both!), you'll have three interviews:
+
+##### Core Values Interview (1 hour)
+We want to know about what you're passionate about, how you worked with teams in the past and why you think 18F is a good fit for you. We expect you to ask the same of us. 
+
+##### Problem Solving Interview (1 hour)
+We'll present you with hypothetical problems and actual problems that we've faced to see how you would handle them. We’re also very curious about your response to problems that you’ve faced in your experience.
+
+##### Technical Interview (1 hour)
+This is where we dive deep into the process, methodologies and tools that you use to get the job done and your beliefs on how teams should work together.
```
Comment (afeld) [inline pages/interview-process.md]: Maybe we should move all the engineering-specific stuff to a separate page?

File: pages/who-we-are-hiring.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+
+We are looking for candidates who are passionate about our mission, with top-notch software development, design, content, and operations skills to match.
+
+## Where Are We Hiring?
+
+Everywhere in the United States! If you have no experience working on remote teams, or are applying for a role that requires in-person interactions with clients, we will ask that you work out of either our San Francisco or D.C. office. However, the majority of our team is distributed across the country, in places like Chicago, New York, Raleigh, Tuscon, Austin, Dayton, Philadelphia, Santa Barbara, Seattle, and Portland.
+
+Our rapidly growing team is home to a variety of roles, including these:
```
Comment (afeld) [inline pages/who-we-are-hiring.md:16]: Maybe add another heading of

``` markdown
# What Roles Are We Hiring?
```

**Comment by elainekamlley** [comment]
Do we think that these are good enough positions to link to for now or should it revert back to the more general buckets from the blog post: https://18f.gsa.gov/2015/02/25/We-Are-Hiring/  until we have that updated language? @cscairns @jamiealbrecht 

@afeld yup. I can add those in and create an issue for content holes.

**Comment by elainekamlley** [comment]
Alrighty! I took another stab at it. Here's a few things I did: 
- Created a collections for interviewing with specific teams. Took out info for engineering and product and created new pages to link out to. 
- @jamiealbrecht confirmed with me that the positions here are the ones to keep until the official ones are finished. 
- fixed typos (but could use other eyes on it) 
- Took out any titles that are incomplete, team Talent can add sections and language when they have capacity. 

@afeld @jamiealbrecht @emileighoutlaw @harrisj  Can anyone take a look? If this is good enough to ship, I'd love to get this on the site by tomorrow.

**Pull request merged**